rts/Lua/LuaUnsyncedRead.cpp

```diff
@@ -3135,8 +3145,14 @@ int LuaUnsyncedRead::GetSoundDevices(lua_State* L)
 
 	for(int i; i < devices.size(); ++i) {
```

Comment:
Isn't `i` uninitialized here? IIRC to zero-initialize it has to be at least `= {}` but perhaps best to just be explicit about it. Also `devices.size()` is `size_t` so perhaps `i` should also be.
```suggestion
	for (size_t i = 0; i < devices.size(); ++i) {
```